StudentsDB/src/main/java/com/musala/simple/students/db/MongoObjectMapper.java

```diff
@@ -0,0 +1,30 @@
+package com.musala.simple.students.db;
+
+import com.google.gson.Gson;
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBCollection;
+import com.mongodb.DBCursor;
+import com.mongodb.DBObject;
+
+public abstract class MongoObjectMapper {
```

Comment:
Here you have to create an interface e.g. ObjectMapper and implement MongoObjectMapper (so that we implement a MySQL object mapper in the future). In the future, if we find out there are mapping similarities between Mongo and MySQL, we will implement an AbstractObjectMapper which provides default mapping and then we will do the custom Mongo/MySQL stuff in the respective mapper that extends the AbstractObjectMapper. ;)